### ISSUE TITLE
WAR task

### ISSUE THREAD

**Issue by rov63rus** [Putnami/putnami-gradle-plugin#62]
Need war task to generate correct WAR that may be deployed on server

**Comment by fdumay** [comment]
Hi @rov63rus,
This plugin depends on the gradle war plugin. When you build the project the jar is generated in 'build/libs'. This war is runnable on any java server.
Do you have any issue with the generated war?

**Comment by rov63rus** [comment]
Yes I have. Resource files that is src/main (*.gwt.xml, *.xml, *properties and etc) are not placed into WEB-INF\classes\. These files are gwt(other frameworks, such as mybatis) specific and important.

So application  without that files is broken.

P.S. I glad this project is alive again.

**Comment by fdumay** [comment]
It's the default gradle behaviour. You must put thoses files in `src/main/resources` to find into your war. Or you can also create a gradle task for copy them.

**Comment by rov63rus** [comment]
I know that. I think this default behavior of "war" task is not user friendly from GWT application  perspective. Then I click "war" task in GWT project I except fully functional war that I can deploy on server. 

That's why I think the plugin need task "gwtWar" to build ready-to-use "war" with one click.
